app/services/post_alerter.rb

```diff
@@ -261,7 +261,6 @@ def should_notify_like?(user, notification)
   end
 
   def should_notify_previous?(user, post, notification, opts)
-    return false unless notification
```

Comment:
why was this removed?

Reply:
I added it here: https://github.com/discourse/discourse/commit/e90f9e5cc47 so basically it is a revert. I only left spec from original commit to ensure I didn't break that edit and add quote thingy